examples/routing_service/routing_service_with_security/signFiles.bat

```diff
@@ -0,0 +1,5 @@
+set OPENSSL_CONF=..\..\..\resources\certAuthority\dsa01\ca\dsa01RootCa.cnf
```

Comment:
ecdsa01, right?

Reply:
Corrected path. I have also added for `rsa01` and `dsa01`, which are commented.